include/seqan3/alphabet/quality/qualified.hpp

```diff
@@ -49,16 +49,20 @@
 namespace seqan3
 {
 
-/*!\brief A seqan3::cartesian_composition that joins a nucleotide alphabet with a quality alphabet.
+/*!\brief Joins an arbitrary alphabet with a quality alphabet.
  * \ingroup quality
- * \tparam sequence_alphabet_t Type of the first letter; must satisfy seqan3::nucleotide_concept.
- * \tparam quality_alphabet_t Types of further letters (up to 4); must satisfy seqan3::quality_concept.
+ * \tparam sequence_alphabet_t Type of the alphabet; must satisfy seqan3::alphabet_concept.
+ * \tparam quality_alphabet_t  Type of the quality; must satisfy seqan3::quality_concept.
  * \implements seqan3::quality_concept
  *
- * This composition pairs a nucleotide alphabet with a quality alphabet. The rank values
- * correpsond to numeric values in the size of the composition, while the character values
- * are taken from the sequence alphabet and the phred values are taken from the quality
- * alphabet.
+ * This composition pairs an arbitrary alphabet with a quality alphabet. Note,
+ * that usually this is a nucleotide alphabet, e. g. a FASTQ sequence stored
```

Comment:
but you are not storing a sequence in a single object here, only two letter :nail_care: